drivers/node/node.go

```diff
@@ -140,6 +140,9 @@ type Driver interface {
 	// RebootNode reboots the given node
 	RebootNode(node Node, options RebootNodeOpts) error
 
+	// RebootNodeAndWaitreboots given node and wait for nodes to be ready.
```

Comment:
nit: `// RebootNodeAndWait reboots a given node and waits for the node to be ready.`

Reply:
DOne